core/primitives-core/src/version.rs

```diff
@@ -222,7 +222,6 @@ impl ProtocolFeature {
             ProtocolFeature::LowerValidatorKickoutPercentForDebugging => 81,
             ProtocolFeature::SingleShardTracking => 82,
             ProtocolFeature::StateWitnessSizeLimit => 83,
-            ProtocolFeature::StatelessnetShuffleShardAssignmentsForChunkProducers => 84,
             ProtocolFeature::PerReceiptHardStorageProofLimit => 85,
```

Comment:
not sure whether shifting the numbers here is safe, so leaving as is.

Reply:
Mocknet and statelessnet will re-start from the same nodes, so I think it is fine (yeah it would not be fine with mainnet/testnet).